contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
+* Multiple business units need a software component.
+* It makes sense for the business units to collaborate; each needs some different changes to the code.
+* One BU doesn't have the resources to do the development for all the rest.
+* InnerSource programs and tools exist.
+
+## Problem  
+The company needs to develop a software product component; multiple business units use this component.
```

Comment:
That sounds more like context to me. Since the BUs are already in agreement to collaborate and given the solution below, is the problem related to the degree of openness of a collaboration (uncontained)?